jina/executors/rankers/__init__.py

```diff
@@ -70,80 +91,25 @@ class Chunk2DocRanker(BaseRanker):
 
     def score(
         self, match_idx: 'np.ndarray', query_chunk_meta: Dict, match_chunk_meta: Dict
-    ) -> 'np.ndarray':
+    ) -> float:
```

Comment:
Instead of passing all the chunks with different parent ids, we assume all the input chunks have the same parent id. In this case, `match_idx[: 0]` is no longer necessary and can be removed.

Reply:
Yes. I like the idea, but I am not sure if it is overkill to delete the first column, should I only select the other 3?

What is the best way to do this with zero runtime cost in numpy?

Reply:
@nan-wang , in this case it is not so easy, because we are creating a numpy array of a custom tuple, so is not easy to access by column. I would go with this solutioin and try to refactor this later, maybe there is some optimization to be done here

Reply:
fair points. Maybe we can add a `TODO` here.